### PR TITLE
Adding details on End of Life process.

### PR DESCRIPTION
**Pull request by fleeting**
This is for #8. Need a review and approval.

I plan to have different section elsewhere about starting a redesign process and creating the initial `redesign` branch, deployment, etc. This is just about handling the end of life.

/cc @defvayne23 @djsaun

### CODE REVIEW

**Comment by defvayne23** [comment]
👍

File: general/git.md [hunk 1]
```diff
@@ -67,18 +69,19 @@ For details on creating, pushing, or checking out tags checkout the [git docs](h
...
 
-**Removing Project**
+**Project Removal**
 * Create a new branch from `master` with the name `EOL-YYYYMMDD` (`EOL-20150220`). Push this new branch to the remote.
 * Open the repo on BitBucket and under settings change the main branch to the new EOL branch you pushed above.
 * Delete any branches that were part of the old site; `master`, `staging`, `dev`, etc. These should be removed from your local and from the remote.
 
 **Project Redesign**
-* Create a new branch from `master` with the name `EOL-YYYYMMDD` (`EOL-20150220`). Push this new branch to the remote.
+* At the start of the redesign process you will create a `redesign` branch. This will allow you to continue to update the live site as needed with the `dev` and `master` branches until the redesign is complete.
```
Comment (fleeting) [inline general/git.md:78]: @defvayne23 Added this line and tweaked the below line to note how to initially start a redesign process.

**Comment by fleeting** [comment]
Pinging @djsaun for approval.

**Comment by djsaun** [comment]
Looks good to me!

**Pull request merged**